packages/terra-i18n/src/terra-dev-site/doc/example/I18nDemo.jsx

```diff
@@ -30,7 +30,18 @@ class Demo extends React.Component {
         locale={this.state.locale}
         messages={this.state.messages}
       >
-        <FormattedMessage id="Terra.ajax.error" />
+        <div>
+          <span> Example Message Format: </span>
+          <FormattedMessage id="Terra.ajax.error" />
+        </div>
+        <p>
+          <span> Example Number Format: </span>
+          <FormattedNumber value={parseFloat('1123432.123')} />
+        </p>
+        <p>
+          <span>Example Date Format: </span>
```

Comment:
Something to note about Intl Formatting, if the locale data is not loaded, it will fallback to the browser default. This is a Intl implementation detail: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation